src/promptflow/promptflow/core/_errors.py

```diff
@@ -0,0 +1,45 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from promptflow.exceptions import ErrorTarget, SystemErrorException, UserErrorException
+
+
+class CoreError(UserErrorException):
+    """SDK base class, target default is CONTROL_PLANE_SDK."""
```

Comment:
why the docstring sys this is an Exception for SDK ?

Reply:
let me change this.